ui/src/app/shared/common.ts

```diff
@@ -38,7 +39,8 @@ export const queryDataTypes: Map<string, FieldDataType> = new Map([
 
 export const queryExtensionsDataTypes: Map<string, FieldDataType> = new Map([
   ['projectId', FieldDataType.Text],
-  ['userId', FieldDataType.Text]
+  ['userId', FieldDataType.Text],
```

Comment:
It feels to me like the `FieldDataType` of a query extensions should be provided as part of the `CapabilitiesResponse` rather than encoded separately in a static UI file. I know I initially added this, and we can for sure leave it to another PR, but do you have any sense of how difficult it would be to refactor the field type into part of the capabilities response?

Reply:
I don't think it would be too difficult.  There's already the concept of type for the `displayFields` part of capabilities. Can you create a new issue for this?

Reply:
Done https://github.com/DataBiosphere/job-manager/issues/483

Reply:
thanks!